go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/newrelic/nri-jmx
+
+go 1.16
+
+require (
+	github.com/kr/pretty v0.2.1
+	github.com/newrelic/infra-integrations-sdk v3.6.7+incompatible
```

Comment:
why v3 instead of v4? 
https://github.com/newrelic/infra-integrations-sdk/blob/master/go.mod#L1

Reply:
I guess it's marked as `incompatible` as v3 didn't specified the gomodules version properly

Reply:
I asked the team and for the moment is not possible to move to v4 since it's too many things to change, and it's planned for the future. 
Regarding the incompatible, go modules requires +incompatible when the dependency has a semver major version of 2, the way to go with go modules in this cases is creating a /v3 instead